modules/installation/pages/hardware-requirements.adoc

```diff
@@ -52,5 +52,15 @@ For {smr} hardware requirements, see xref:retail:retail-requirements.adoc[].
 |                          | _Production Server_ Minimum 8{nbsp}GB
 | Disk Space:              | [path]``/`` _(root)_ Minimum 24{nbsp}GB
 |                          | [path]``/srv`` Minimum 100{nbsp}GB
-|                          | [path]``/var/cache`` _(Squid)_ Minimum 100{nbsp}GB
+|                          | [path]``/var/cache`` _(Squid)_ Recommended: minimum 100{nbsp}GB
 |===
+
+{productname} Server stores packages in [path]``/var/spacewalk``. In case it runs out of disk space, the repository synchronization will fail. Based on what and how many client operating systems and repositories you plan to mirror, you shold estimate how much space to assign to [path]``/var/spacewalk`` and provide enough space.
+
+{productname} Proxy stores packages in [path]``/var/cache`` but only as a cache. When not enough space is available under [path]``/var/cache``, {productname} Proxy will remove old, unused packages and replace them with newer packages.
+
+Because of that:
+
+* The larger [path]``/var/cache`` is on the {productname} Proxy or {productname} Retail Branch Server, the less traffic there will be between it and the {productname} Server. 
+* By making [path]``/var/cache`` on the {productname} Proxy at least the same size as [path]``/var/spacewalk`` in the {productname} Server, you could avoid a lot of traffic after the first sync.
+* [path]``/var/cache`` can be rather small on the {productname} Server compared to {productname} Proxy (less than 15G should be enough).
```

Comment:
```suggestion
{productname} Server stores packages in the [path]``/var/spacewalk/`` directory. 
Repository synchronization fails if this directory runs out of disk space. 
You can estimate how much space the [path]``/var/spacewalk/`` directory requires based on the number and type of clients and repositories you plan to mirror.

{productname} Proxy caches packages in the [path]``/var/cache/`` directory.
If there is not enough space available in [path]``/var/cache/``, the proxy will remove old, unused packages and replace them with newer packages.

As a result of this behavior:

* The larger [path]``/var/cache/`` directory is on the {productname} Proxy, the less traffic there will be between it and the {productname} Server. 
* By making the [path]``/var/cache/`` directory on the {productname} Proxy at least the same size as [path]``/var/spacewalk/`` in the {productname} Server, you avoid a large amount of traffic after the first synchronization.
* The [path]``/var/cache/`` directory can be small on the {productname} Server compared to {productname} Proxy.
We recommend less than 15{nbsp}GB in this situation.
```

Reply:
Instead of the last sentence (`We recommend less than 15{nbsp}GB in this situation.`), I'd write for clarity:
```
{suse} recommends less than 15{nbsp}GB on the{productname} Server in this situation.
```
And let's avoid "we" phrases.

Reply:
I agree to most of @Loquacity 's changes. I'm not sure less than 15 GB are actually "recommended", it's actually "allowed", "enough", etc.
@mcalmer what do you think?

Reply:
It depends on how many repos you sync. On a ISS master additional space is required as all data gets pre-generated also in /var/cache. 
I think "less than 15 GB" is wrong in some cases. Especially when the customer sync a lot of channels.

Reply:
How much additional space would be required on an ISS master? 

Can we give some kind of a metric, e. g. "each channel requires X MB of disk on /var/cache on a normal Server and Y MB additionally if ISS master"?

Reply:
~ 80M -100M per channel. RES channel 1G.
Double the number when Server is an ISS master.
(just guessing by looking at some numbers on our server)

Reply:
I have updated the text with Lana's and MC's suggestions.